CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Changelog
 
+## 10.2.0
+
+This release of Teleport contains multiple improvements and bug fixes.
+
+* Fixed issue with `tsh` reporting "no suitable devices found" when using Yubikeys. [#16011](https://github.com/gravitational/teleport/pull/16011)
+* Fixed potential panic when using unsupported Google application credentials file. [#16042](https://github.com/gravitational/teleport/pull/16042)
+* Fixed issue with database connections getting terminated due to idle timeout when running long queries. [#16017](https://github.com/gravitational/teleport/pull/16017)
+* Fixed issue with shared directory operations terminating Desktop Access sessions. [#16054](https://github.com/gravitational/teleport/pull/16054)
```

Comment:
I think we can remove this one. Directory sharing is new to this release, so any issues that it fixed were only present on master.

Reply:
Instead of removing it, let's just say "Add support for directory sharing to desktop access."